<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(53, 53, 79);
border: 1px solid #00aaff; /* Contorno neon azul */
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTableWidget" name="purchaseTable">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>441</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
	color: rgb(201, 255, 252);
    background-color: #d9fffb;; /* Fondo de la tabla (celeste muy tenue) */
}

QTableWidget QHeaderView:section {
    background-color: #a9bdff ; /* Fondo del encabezado de la columna (celeste muy tenue) */
    color: black; /* Color del texto del encabezado de la columna (negro) */
}

QTableWidget::item {
    padding: 5px; /* Espaciado interno de las celdas */ 
	color: black;
}

QTableWidget::item:selected {
    background-color: #d1d1d1; /* Fondo de la celda seleccionada (celeste muy tenue) */
    color: black; /* Color del texto de la celda seleccionada */
}</string>
   </property>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Producto</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Categoria</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
border: 0px solid #00aaff;</string>
   </property>
   <property name="text">
    <string>Cliente:</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
border: 0px solid #00aaff;</string>
   </property>
   <property name="text">
    <string>Fecha:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
border: 0px solid #00aaff;</string>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
border: 0px solid #00aaff;</string>
   </property>
   <property name="text">
    <string>Total:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="clientComboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 103, 155);
font: 87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
border-radius: 6px;
border: 0px solid black;</string>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 103, 155);
font: 87 12pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
border-radius: 6px;
border: 0px solid black;</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
